### PR TITLE
Fix swiping

### PR DESCRIPTION
**Pull request by erlingfn**
Bump react-swipeable to v6.2.1 and fix swiping in the process.

Closes: #2574

### CODE REVIEW

**Review by Arashfa0301 — APPROVED** [review]
LGTM

**Review by SmithPeder — APPROVED** [review]
lgTM

File: app/routes/photos/components/GalleryPictureModal.js [hunk 1]
```diff
@@ -111,6 +111,16 @@ const RenderGalleryPicture = ({
   </div>
 );
 
+function Swipeable(props: {
```
Comment (ivarnakken) [inline app/routes/photos/components/GalleryPictureModal.js:114]: Why no arrow function? 😢

Comment (erlingfn) [reply app/routes/photos/components/GalleryPictureModal.js:114]: If it how we do it for the rest of the project, I could change it. Yet I personally prefer the traditional function syntax for these types of functions.

File: app/routes/photos/components/GalleryPictureModal.js [hunk 2]
```diff
@@ -111,6 +111,16 @@ const RenderGalleryPicture = ({
   </div>
 );
 
+const Swipeable = (props: {
```
Comment (eikhr) [inline app/routes/photos/components/GalleryPictureModal.js:114]: arrow functions are cool 😎

**Review by eikhr — APPROVED** [review]
LGTM

**Pull request merged**